miden-tx/tests/integration/scripts/p2id.rs

```diff
@@ -175,9 +175,11 @@ fn p2id_script_multiple_assets() {
         )
         .unwrap();
 
+    let tx_args_target = TransactionArgs::new(Some(tx_script_target), None);
+
     // Execute the transaction and get the witness
-    let executed_transaction = executor
-        .execute_transaction(target_account_id, block_ref, &note_ids, Some(tx_script_target))
+    let transaction_result = executor
```

Comment:
I don't think we need to change `executed_transaction` into `transaction_result` here and in other places in this file.